server/services/netatmo/index.js

```diff
@@ -0,0 +1,34 @@
+const logger = require('../../utils/logger');
+const NetatmoManager = require('./lib');
+const NetatmoController = require('./api/netatmo.controller');
+
+module.exports = function NetatmoService(gladys, serviceId) {
+  const netatmoManager = new NetatmoManager(gladys, serviceId);
+  /**
+   * @public
+   * @description This function start netatmo service and connect it
+   * @example
+   * gladys.services.netatmo.start();
+   */
+  async function start() {
+    logger.log('Starting netatmo service');
+    netatmoManager.connect();
+  }
+
+  /**
+   * @public
+   * @description This function stops the service
+   * @example
+   *  gladys.services.netatmo.stop();
+   */
+  async function stop() {
+    logger.log('Stopping Netatmo service');
```

Comment:
Same here ( Can you use logger.info instead ? )

Reply:
It's good !!